### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by wolfe**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "name": "django_base",
     "version": "0.0.0",
     "dependencies": {
-        "bootstrap": "2.2.1",
-        "jquery": "1.8.3",
+        "bootstrap": "2.3.2",
+        "jquery": "2.0.2",
```
Comment (pmdarrow) [inline bower.json]: jQuery 2.x drops IE8 support, just FYI. Might want to check with John if we're still targeting that and if so, use 1.10.x.

Comment (wolfe) [inline bower.json]: Hmm --- turns out bootstrap on 1.8 --- or is it 1.9 --- I'm inclined to match it.  (Or omit requirement altogether.)

David

**Comment by pmdarrow** [comment]
LGTM.

**Comment by thurloat** [comment]
did you not end up using the .bowerrc file?

**Comment by wolfe** [comment]
Oops --- forgot about bowerrc.  Had bugs.

File: setup [hunk 2]
```diff
@@ -51,27 +55,33 @@ do
...
+mkvirtualenv --python=python2.7  --no-site-packages $PROJECT
 pip install -r requirements.txt -f https://s3.amazonaws.com/sheepdog-assets/feta/index.html --no-index
 npm install
 
 printf 'export PATH=%s/node_modules/.bin:$PATH\n' `pwd` >> $VIRTUAL_ENV/bin/postactivate
 
-ln -s $VIRTUAL_ENV .env
-ln -s $VIRTUAL_ENV/lib/python2.7/site-packages/ .env-packages
-ln -s $VIRTUAL_ENV/lib/python2.7/site-packages/django/ .env-django
+# Warning:  If node is too new a version, you'll get an error; not sure why
```
Comment (pmdarrow) [inline setup:81]: Figured out why this was happening. It's not because node is too new - you had a syntax error in bower.json :)

**Pull request merged**